From 3069b953dd1303eb4bd1171a8f5c5501ecdafc29 Mon Sep 17 00:00:00 2001
From: Vassil Vassilev <v.g.vassilev@gmail.com>
Date: Sun, 3 Mar 2024 20:32:15 +0000
Subject: [PATCH] [cmake] Work around a bug in the llvm config.

In short we use variables which require including `GNUInstallDirs` but we are
expecting somebody else to include it for us.

See llvm/llvm-project#83802
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 548e535..f4efb0e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.13)

+include(GNUInstallDirs)
+
 set(CMAKE_MODULE_PATH
   ${CMAKE_MODULE_PATH}
   "${CMAKE_CURRENT_SOURCE_DIR}/cmake"
--
2.37.1 (Apple Git-137.1)
